frontend/src/App.js

```diff
@@ -0,0 +1,15 @@
+import Register from "./components/register";
+import Navbar from "./components/navbar";
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar />
+      <div className="container mt-5">
+        <Register />
```

Comment:
Not only register page will be implemented, so get ready your file structure and routes